app/importers/trade_lead/fbopen_importer.rb

```diff
@@ -0,0 +1,13 @@
+# For shared methods used by the two FBO importers
+module TradeLead
+  module FbopenImporter
+    def process_geo_fields(entry)
+      entry[:country] = IsoCountryCodes.find(entry[:country]).alpha2 rescue nil
+      entry.merge! add_geo_fields([entry[:country]])
+    end
+
+    def extract_end_date(entry)
+      [entry[:arch_date], entry[:resp_date]].reject(&:nil?).max
+    end
+  end
+end
```

Comment:
Ha, yeah not sure either why what worked, maybe there's a clear enough delineation between namespacing and mixins to where Ruby doesn't get confused by this.